script.js

```diff
@@ -2,6 +2,151 @@ const menu = document.querySelector('.menu');
 const menuList = document.querySelector('.t-menu');
 const close = document.querySelector('.c-button');
 const lists = document.querySelector('.tul');
+const allCards = document.querySelector('.all-cards-container');
+const section = document.createElement('section');
+allCards.appendChild(section);
+section.innerHTML = `
+    <section id="tul-1" class="middle-section-container">
+      <div class="image-area">
+        <img src="photo/logos/Snapshoot-Portfolio.png" alt="Benjamin">
+      </div>
+
+      <div class="destop-tonic-img">
+        <img src="photo/navs/Snapshoot-Portfolio.jpg" alt="">
+      </div>
+
+      <div class="hero">
+        <div class="tonic">
+          <h1>Tonic</h1>
+            <div class="canopy-wrapper">
+              <p>CANOPY</p>
+                <ul>
+                <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+            </div>
+        </div>
+
+          <div class="text-area">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+          </div>
```

Comment:
- [ ] `required`
- [ ] Great job displaying content dynamically 👏 
- [ ] however, the Modal popup data is supposed to be populated with the data from the projects array.
- [ ] your code should look something like the screenshot below:
![image](https://user-images.githubusercontent.com/66230306/181616800-21990ea2-e6cd-4bac-a269-1ffb82b61a74.png)
`please make adustments`